lib/ssh_scan/stats.rb

```diff
@@ -0,0 +1,43 @@
+module SSHScan
+  class Stats
+    def initialize
+      @requests = []
+    end
+
+    def new_scan_request
+      @requests << Time.now
+    end
+
+    def size
+      @requests.size
+    end
```

Comment:
I guess you could also just add a method to say "give me all stats" as a hash and then just JSONify that and make the /api/stats route a one-liner :)